src/Browsershot.php

```diff
@@ -254,6 +257,26 @@ public function setProxyServer(string $proxyServer)
         return $this;
     }
 
+    public static function jsLinksInWhitelist(string $html, $safeDomains = []) {
+        // https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#php
+        libxml_set_external_entity_loader(null);
+
+        $dom = new \DOMDocument();
+        $dom->loadHTML($html);
+
+        $xpath = new \DOMXPath($dom);
+
+        foreach($xpath->query("//script[@src]") as $queryResult) {
+            $untrustedJSLink = $queryResult->getAttribute("src");
+
+            foreach ($safeDomains as $safeDomain) {
+                if($untrustedJSLink !== $safeDomain) {
+                    throw JSLinkIsNotWhitelisted::make();
```

Comment:
Pass the `jsLink` to the exception and let the exception display that link, so users know which link is not allowed.